src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal.js

```diff
@@ -207,9 +207,9 @@ export const CloudEnvironmentModal = ({
           shape: 'pause',
           toolLabel,
           onClick: () => stopApp(toolLabel),
-          disabled: busy || !canCompute,
+          disabled: cloudProvider === cloudProviderTypes.AZURE || busy || !canCompute,
```

Comment:
Add a TODO comment to these lines with the ticket number for the deallocate work - I think it's IA-3993. That will be when we can enable pausing.

Reply:
Can we not leverage the `isPauseUnsupported` flag for this? On  line 206, for other tools that don't support pause, we hide this icon entirely

Reply:
Ah, I guess not if we want to show a tooltip...

Reply:
I think the AC here is not to hide the button but to disable it with the note that the functionality isn't available yet